unittest/CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ IF(BUILD_PYTHON_INTERFACE)
 
   TARGET_LINK_LIBRARIES(python_parser ${PYWRAP})
   TARGET_LINK_LIBRARIES(python_parser ${PYTHON_LIBRARIES})
+  SET(PYTHONPATH "${CMAKE_BINARY_DIR}/bindings/python")
+  IF(DEFINED ENV{PYTHONPATH})
```

Comment:
You are not obliged to add this IF condition here

Reply:
Do you suggest to remove `IF(DEFINED ENV{PYTHONPATH})` and `ENDIF(DEFINED ENV{PYTHONPATH})` but keep `SET(PYTHONPATH "${PYTHONPATH}:$ENV{PYTHONPATH}")`, or also remove this one ?

for reference: https://github.com/jrl-umi3218/jrl-cmakemodules/pull/137#discussion_r191808174

Reply:
Just keep as you mentioned, it is sufficient. Or I may be wrong?

Reply:
Well… In fact, this `IF` was meant to prevent having a trailing `:` at the end of the `PYTHONPATH`, as it would add the current working directory in `sys.path`. 
But firstly I am not even sure if this can raise issues, and secondly this `IF` is useless when `ENV{PYTHONPATH}` is defined but empty.

So let's do what you suggest, and if someone find a real case issue with that, we well have to use `IF(ENV{PYTHONPATH})`, and put that also in the cmake submodule.

Reply:
done in 3c677514bdb578a69e211e59c55c6c86b68fb85f